pkg/cli/mirror/list/releases.go

```diff
@@ -22,6 +28,13 @@ type ReleasesOptions struct {
 	Version  string
 }
 
+type ReleaseVersion struct {
```

Comment:
One more nit: This is public and could use a comment.

Reply:
actually, this shouldn't be public. Adding comment and making private